_includes/manuals/1.10/4.3.6_smartproxy_puppet.md

```diff
@@ -124,6 +124,58 @@ The puppetssh provider uses SSH to connect to the client using SSH keys and run
 #:puppetssh_keyfile: /etc/foreman-proxy/id_rsa
 </pre>
 
+Here is a quick howto to enable puppetssh on your server.
+
+On the `foreman-proxy server`, enable the puppetssh provider and uncomment user and ssh keys settings:
+<pre>
+sed -i 's!^#\?\(:puppet_provider:\).*$!\1 puppetssh!' \
+  /etc/foreman-proxy/settings.d/puppet.yml
+sed -i 's!^#\?\(:puppetssh_user:\).*$!\1 root!' \
+  /etc/foreman-proxy/settings.d/puppet.yml
+sed -i 's!^#\?\(:puppetssh_keyfile:\).*$!\1 /etc/foreman-proxy/id_rs!' \
+  /etc/foreman-proxy/settings.d/puppet.yml
+</pre>
+
+We also need to actually generate this new SSH key:
```

Comment:
ditto about pronouns, this could be written as "The new SSH key needs to be generated by running:"